test/libsolidity/syntaxTests/viewPureChecker/inline_assembly_instructions_allowed_view.sol

```diff
@@ -73,7 +73,7 @@ contract C {
             pop(coinbase())
             pop(timestamp())
             pop(number())
-            pop(difficulty())
+            pop(prevrandao())
```

Comment:
Maybe it's better to have two tests here, one `<paris` and one `>= paris`?

Reply:
I introduced another test [inline_assembly_instructions_allowed_view_london.sol](https://github.com/ethereum/solidity/pull/13531/files#diff-11182a2286915423c63070802c2a16488b8f2e4f155dc69243d2567eca086a3a) for london that contains only the changed opcode